COPYRIGHT.md

```diff
@@ -0,0 +1 @@
+Copyright (c) 2015-2018, Zend Technologies USA, Inc.
```

Comment:
Needs the following line after it:

```text
All rights reserved.
```

The start year on this one should probably be 2005, as it references the project, and not necessarily the repository.